dotnet/.ci/dotnet-cloud-run-hello-world.cloudbuild.yaml

```diff
@@ -1,25 +1,36 @@
 steps:
+- id: Generate unique service name
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    random-string() {
+        cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-z0-9' | fold -w 15 | head -n 1
```

Comment:
Can be shortened to `cat /dev/urandom | LC_CTYPE=C tr -dc 'a-z0-9' | head -c 15`